frontend/src/components/common/header/header.module.scss

```
@@ -0,0 +1,113 @@
+a {
+  color: #000;
+}
+
+.container {
+  position: absolute;
```

```suggestion
  position: sticky;
```
https://developer.mozilla.org/en-US/docs/Web/CSS/position